kong/reports.lua

```diff
@@ -403,6 +410,10 @@ return {
     local suffix, err = get_current_suffix()
     if suffix then
       incr_counter(count_key .. ":" .. suffix)
+
+      if ngx.ctx.ran_go_plugin then
```

Comment:
Same here, the `log` function should be changed to pass in `ctx` instead of using `ngx.ctx` directly, to keep the style among codes consistent.